### PR TITLE
docs: add `getAllViews` `getAllTypes` `dropAllViews` and `dropAllTypes`

### PR DESCRIPTION
**Pull request by Julien-R44**
Related PR : https://github.com/adonisjs/lucid/pull/784

### CODE REVIEW

File: content/reference/database/query-client.md [hunk 1]
```diff
@@ -114,6 +114,28 @@ console.log(tables)
...
+
+```ts
+const views = await client.getAllViews()
+console.log(views)
+```
+
+---
+
+### getAllTypes
+⚠️ Postgres/Redshift only
```
Comment (RomainLanz) [inline content/reference/database/query-client.md]: I am not sure about the emoji usage here.

File: content/reference/database/query-client.md [hunk 2]
```diff
@@ -124,6 +146,32 @@ console.log(columns)
...
+await client.dropAllViews()
+
+// specify schemas ( for Postgres and Redshift )
+await client.dropAllViews(['public'])
+```
+
+---
+
+### dropAllTypes
+⚠️ Postgres/Redshift only
```
Comment (RomainLanz) [inline content/reference/database/query-client.md]: Same.

Comment (Julien-R44) [reply content/reference/database/query-client.md]: Fixed !

File: content/reference/database/query-client.md [hunk 3]
```diff
@@ -125,7 +125,7 @@ console.log(views)
 ---
 
 ### getAllTypes
-⚠️ Postgres/Redshift only
+For Postgres/Redshift only
```
Comment (thetutlage) [inline content/reference/database/query-client.md]: A small thing. I will keep it as follows:

```
Returns an array of all the database custom types. The method works only with **Postgres and Redshift**
```

File: content/reference/database/query-client.md [hunk 4]
```diff
@@ -114,6 +114,26 @@ console.log(tables)
...
+
+```ts
+const views = await client.getAllViews()
+console.log(views)
+```
+
+---
+
+### getAllTypes
+Returns an array of all the database custom types. The method works only with **Postgres and Redshift**
```
Comment (RomainLanz) [inline content/reference/database/query-client.md]:
```suggestion
Returns an array of all the database custom types. The method works only with **Postgres and Redshift**.
```

File: content/reference/database/query-client.md [hunk 5]
```diff
@@ -124,6 +144,27 @@ console.log(columns)
...
+await client.dropAllViews()
+
+// specify schemas ( for Postgres and Redshift )
+await client.dropAllViews(['public'])
+```
+
+---
+
+### dropAllTypes
+Drop all custom types inside the database. The method works only with **Postgres and Redshift**
```
Comment (RomainLanz) [inline content/reference/database/query-client.md]:
```suggestion
Drop all custom types inside the database. The method works only with **Postgres and Redshift**.
```

Comment (Julien-R44) [reply content/reference/database/query-client.md]: Thanks !

**Comment by thetutlage** [comment]
Seems like there are conflicts in the commit caused by https://github.com/adonisjs/docs.adonisjs.com/pull/117. Can you please rebase `develop` branch and then push

**Comment by thetutlage** [comment]
Thanks :)

**Pull request merged**